cassettes/test_failed_connectivity.yaml

```diff
@@ -16,7 +16,7 @@ interactions:
       User-Agent:
       - python-requests/2.32.3
     method: POST
-    uri: https://notauri.edu/api/oauth
+    uri: https://cleaned.example.edu/api/oauth
```

Comment:
Since we don't generally record this cassette, I just updated it by hand to match the updated cleaner behavior.